t/lib/OpenQA/Test/Utils.pm

```diff
@@ -244,7 +244,6 @@ sub create_websocket_server {
         use OpenQA::WebSockets::Controller::Worker;
         use OpenQA::WebSockets::Plugin::Helpers;
 
-        # TODO: Kill it with fire!
```

Comment:
I strongly object to this change! 😈

Reply:
Can you please be a bit more constructive here? I do not know what you want. I would like to remove a TODO comment in code which I consider offensive and not helpful. Also it has been here for multiple years and unchanged so obviously the comment does not help.

Reply:
I realized this comment is already gone with another PR merged